### PR TITLE
Initial draft

### PR DESCRIPTION
**Pull request by vincentneko**
This add another option for a generation 1 draft, as suggested by @ssimeonoff 
All players are dealt 10 cards.
They pick one and pass the rest clockwise.
Draft ends when your cards are back in your hand.

Corporations and prelude dealt are visible during draft phase.

### CODE REVIEW

File: src/Game.ts [hunk 1]
```diff
@@ -638,19 +661,34 @@ export class Game implements ILoadable<SerializedGame, Game> {
...
           let cards = this.unDraftedCards.get(this.getDraftCardsFrom(player));
           this.unDraftedCards.delete(this.getDraftCardsFrom(player));
-          player.runDraftPhase(this, this.getNextDraft(player).name, cards);
+          player.runDraftPhase(initialDraft, this, this.getNextDraft(player).name, cards);
         }
       });
     }
 
+    /*
+    private runInitialDraftRound(): void {
```
Comment (bafolts) [inline src/Game.ts:678]: If this isn't being used it should be removed.

File: tests/Turmoil.spec.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ describe("Turmoil", function () {
         const player = new Player("test", Color.BLUE, false);
         const gameOptions = {
             draftVariant: false,
+	    initialDraftVariant: false,
```
Comment (bafolts) [inline tests/Turmoil.spec.ts:17]: There are tabs included here and not spaces which throws off the alignment.

**Comment by alrusdi** [comment]
@vincentneko can you please resolve the merging-with-master-branch conflict ?

**Comment by vincentneko** [comment]
> @vincentneko can you please resolve the merging-with-master-branch conflict ?

Done

**Comment by alrusdi** [comment]
@bafolts can we please you to review it again? This is amazing feature. We really need it :-)

**Pull request merged**